src/util/util_string.h

```diff
@@ -21,9 +25,18 @@ namespace dxvk::str {
     format1(stream, args...);
     return stream.str();
   }
-  
+#ifdef __WINE__
+  inline std::string fromws(const std::wstring& ws) {
+    char *ret = nullptr;
+    if (ws.size() > 0) {
+        ret = new char[ws.size() + 1];
+        WideCharToMultiByte( CP_UTF8, 0, ws.c_str(), -1, ret, ws.size() + 1, NULL, NULL );
+    }
+    return ret ? ret : "";
+  }
```

Comment:
Why is this needed? Also, you're leaking memory.

Reply:
Current `std::codecvt_utf8` conversion in `fromws` don't work with wine, maybe it should be changed to use different encoding type (UTF-16 ?), but with lack of knowledge I'm trying to do my best to make this conversion work with winegcc generated libraries.

Also std::codecvt_* are marked as deprecated in C++17, maybe something like shown in [example](http://en.cppreference.com/w/cpp/locale/codecvt) should be used to work for both MinGW and winebuild.